app/assets/stylesheets/_main.scss

```diff
@@ -0,0 +1,43 @@
+.main {
+  margin: 46px auto 0;
+  width: 325px;
+
+  h1 {
+    font-size: 1.8rem;
+    font-weight: bold;
+    margin: 0 0 1.8rem;
+  }
+
+  h2 {
+    font-size: 1.6rem;
+    font-weight: bold;
+    margin: 0;
+  }
+
+  h3 {
```

Comment:
Do we want all of these styling for `.main` only? Should they be put in a separate `_typography.scss` file?